CanvasColorSpaceProposal.md

```diff
@@ -139,14 +138,23 @@ else
 
 #### WebGL behavior
 
-* Values stored in WebGL backbuffers are in the canvas' color space.
-* Values written by ``gl_FragColor`` use the primaries of the canvas' color space.
-* The color encoding function is defined as follows:
-* For formats of ``"unorm8"``, ``"unorm16"``, and ``"float16"`` the color encoding function is:
+Values stored in WebGL backbuffers are in the canvas' color space.
+This includes values accessed directly (e.g, through `ReadPixels`) and all values written via shaders in the graphics pipeline.
+
+The context attribute ``framebufferColorEncodingSRGB`` controls the color encoding function applied to values assigned to ``gl_FragColor`` after the blending stage of the graphics pipeline.
+* If is set to ``false`` then:
+  * The behavior is the same as when ``GL_FRAMEBUFFER_ATTACHMENT_COLOR_ENCODING`` is ``GL_LINEAR`` (as described in section 17.3.9 of the [OpenGL 4.3 specification](https://www.khronos.org/registry/OpenGL/specs/gl/glspec43.core.pdf)).
```

Comment:
WebGL is based on the OpenGL ES specifications. Could you please reference section 6.1.13 "Framebuffer Object Queries" of the ES 3.0.6 spec, https://www.khronos.org/registry/OpenGL/specs/es/3.0/es_spec_3.0.pdf ?

Reply:
Done, but I link to section 4.1.8 "sRGB Conversion", because that's where the behavior is defined.

Also changed this from "is ``GL_LINEAR``" to "is not ``SRGB``", because that's the language of the spec.